src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -622,6 +630,8 @@ public boolean isParticipantIdentifierInstructor(String participantIdentifier) {
                 possibleRecipients = getSortedListOfTeams();
                 break;
             case NONE:
+                possibleRecipients = new ArrayList<String>(); 
+                possibleRecipients.add(Const.USER_NOBODY_TEXT);
                 break;
             default:
```

Comment:
If the default cases in all these switch statements are expected to never be reached, maybe we can add a severe log so that we can catch missed cases like this in the future.